bundles/subclipse.ui/src/org/tigris/subversion/subclipse/ui/actions/SVNPluginAction.java

```diff
@@ -90,9 +90,6 @@ public SVNPluginAction(IConfigurationElement element) {
         setImageDescriptor(desc);
       }
     }
-
-    // Give delegate a chance to adjust enable state
-    selectionChanged(StructuredSelection.EMPTY);
```

Comment:
This change removes the selectionChanged() call from the alternateCompareActions as well, but from what I can tell they don't need to be initialized to empty. It's a simple loop to add calling selectionChanged() back for the alternateCompareActions if you think it's necessary.